src/admin/index.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express'
+const adminRouter = express.Router()
+
+adminRouter.use('/*', (req, res, next) =>  {
+  if (!req.cookies.adminPass) return res.redirect('/login')
+  next()
+})
+
+adminRouter.get('/serviceHostKeys', (req, res) => {
+  res.render('admin')
```

Comment:
I would create a folder called `admin`, and then call this view `admin/serviceHostKeys`